packages/fast-permutator/__tests__/test-utils/index.js

```diff
@@ -0,0 +1,283 @@
+var get = require('lodash').get,
+    permutator = require('../../src/index'),
+    refSchemas = [];
+
+function prettyJSON(obj) {
+    return JSON.stringify(obj, null, 2);
+}
+
+/**
+ * Exported method for getting shallow refs
+ */
+
+describe('Reference IDs', function() {
+    let possibleRefs;
+
+    beforeAll((done) => { // done
+        let schemaData = require('../schemas/primitive/ref.schema.json');
```

Comment:
does this really need to be async? It doesn't look like you're calling done based on any async process. Same for other `beforeAll` functions.